mapswipe_workers/mapswipe_workers/utils/api_calls.py

```diff
@@ -25,12 +97,17 @@ def ohsome(request: dict, area: str, properties=None):
     logger.info("Filter: " + request["filter"])
     response = requests.post(url, data=data)
     if response.status_code != 200:
-        err = f"ohsome Request failed: {response.status_code}"
+        err = f"ohsome request failed: {response.status_code}"
         logger.warning(
             f"{err} - check for errors in filter or geometries - {request['filter']}"
         )
         logger.warning(response.json())
         raise CustomError(err)
     else:
         logger.info("Query succesfull.")
-    return response.json()
+
+    response = response.json()
+
+    if properties:
+        remove_noise_and_add_user_info(response)
```

Comment:
this function should return something, or not?

Reply:
good call for readability; worked and not caused any errors because dict is immutable, so it does not have to be returned